docs.md

```diff
@@ -6,6 +6,6 @@ description: |
     Pico is a flat file CMS, this means there is no administration backend or database to deal with.<br />
     You simply create .md files in the "content" folder and that becomes a page.
 collection: docs
-nav: 2
+nav: 3
```

Comment:
I know why `3` was missing originally. `development.md` is `3`, but the page is hidden at the moment anyway.